File: aws/AWS_Lambda_in_Action/Chapter14/updateFunctionCode.py
awslambda = boto3.client('lambda')

response = awslambda.update_function_code(
    FunctionName='anotherGreetingsOnDemand',
    ZipFile=b'bytes',
    S3Bucket='danilop-functions',
    S3Key='code/greetingsOnDemand-v2.zip',
    Publish=True
)

File: aws/AWS_Lambda_in_Action/Chapter14/updateFunctionCode.js
var lambda = new AWS.Lambda();
var params = {
    FunctionName: 'anotherGreetingsOnDemand',
    Publish: true,
    S3Bucket: 'danilop-functions',
    S3Key: 'code/greetingsOnDemand-v2.zip'
};
lambda.updateFunctionCode(params, function(err, data) {
    if (err) console.log(err, err.stack);
    else console.log(data);
});
